### PR TITLE
Problem: Need to complete logs in diagnostic archive

### PR DESCRIPTION
**Pull request by aquette**
Solution: Add automation audit logs

Signed-off-by: Arnaud Quette <ArnaudQuette@Eaton.com>

### CODE REVIEW

**Comment by aquette** [comment]
@jimklimov  and @MimiXav this one is mergeable as is, without the need to wait for the other PRs. It will simply end up as a noop until /run/etn-envvars.env is available (with KARAF_LOG set)

File: tools/diagnostic-information [hunk 1]
```diff
@@ -131,6 +137,19 @@ system_logs() {
     cp -H -f /var/log/tntnet/www-audit* "${WD}/${TMPDIR}/logs/"
 }
 
+ipm_logs() {
+    verbose "ipm logs..."
+    # FIXME: check for activemq logs
+    if [ -d "$KARAF_LOG" ]
```
Comment (jimklimov) [inline tools/diagnostic-information]: For the case there is no `etn-envvars.env` (e.g. its service not enabled), check also for not-emptiness of `$KARAF_LOG` variable expansion.

On similar note, pack also nginx logs?

Comment (aquette) [reply tools/diagnostic-information]: well, I asked the same to myself for -d Vs -z, and the fact is that empty KARAF_LOG results in the same for the -d test. Still, fixed, thx

Comment (jimklimov) [reply tools/diagnostic-information]: resloved?... commit not pushed up yet? :)

**Pull request merged**